go.mod

```diff
@@ -11,14 +11,16 @@ require (
 	github.com/cenkalti/backoff/v4 v4.3.0
 	github.com/docker/go-units v0.5.0
 	github.com/dolmen-go/contextio v0.0.0-20200217195037-68fc5150bcd5
+	github.com/elastic/beats/v7 v7.17.20
```

Comment:
This should not be here. Happens sometimes when the build directory has extracted beats in it. This needs to be cleaned up before merging.

Reply:
Thanks @blakerouse, I cleaned /bin folder and made the update.
Seems that not got passed and check-ci does not fail

(The failures are not related with this changes I think)

Reply:
@blakerouse I have tried to re-run the build with no luck. 
I can see lots of `ResourceExistsError `

Also for the SonarQube Analysis do you have a suggestion?